docs/admin/logs/_index.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "Logging"
+weight: 50
+type: "docs"
+aliases:
+  - /docs/install/collecting-logs
+showlandingtoc: "false"
+---
+
+You can use [Fluent Bit](https://docs.fluentbit.io/), a log processor and forwarder, to collect your kubernetes logs in a central directory.
+
+This is not required to run Knative, but can be helpful with [Knative Serving](../serving), which will automatically delete pods and associated logs when they are no longer needed.
+
+Fluent Bit supports exporting to a number of other log providers. If you already have an existing log provider, for example, Splunk, Datadog, ElasticSearch, or Stackdriver, you can follow the [FluentBit documentation](https://docs.fluentbit.io/manual/pipeline/outputs) to configure log forwarders.
+
+Setting up log collection requires two steps:
+
+1. Running a log forwarding DaemonSet on each node.
+2. Running a collector somewhere in the cluster.
+
+In the following example, a StatefulSet is used, which stores logs on a Kubernetes PersistentVolumeClaim, but you can also use a HostPath.
+
+## Setting up the collector
+
+Set up the collector before the forwarders. You will need the address of the collector when configuring the forwarders, and the forwarders may queue logs until the collector is ready.
+
+![System diagram: forwarders and co-located collector and nginx](system.svg)
+
+<!-- yuml.me UML rendering of:
+[Forwarder1]logs->[Collector]
+[Forwarder2]logs->[Collector]
+
+// Add notes
+[Collector]->[shared volume]
+[nginx]-[shared volume]
+-->
+
+The [`fluent-bit-collector.yaml`](./fluent-bit-collector.yaml) defines a
+StatefulSet, as well as a Kubernetes Service which allows accessing and reading
+the logs from within the cluster. The supplied configuration will create the
+monitoring configuration in a namespace called `logging`.
+
+You can apply the configuration by entering the command:
+
+```shell
+kubectl apply --filename https://github.com/knative/docs/raw/main/docs/install/collecting-logs/fluent-bit-collector.yaml
+```
+
+The default configuration will classify logs into:
+
+- Knative apps, or pods with an `app=Knative` label
```

Comment:
Should apps be "services" here?

Reply:
I think it's the same difference but I didn't write this, I'd imagine @skonto did maybe, I just cleaned it up, so let's get his opinion before we change any wording relating to components?

Reply:
I didnt write this part but I think using `services` is more precise, although this label is used in different places and seems a bit misleading eg. serving controller.